docs/integration/alerts-webhooks-custom-params.md

```diff
@@ -0,0 +1,21 @@
+title: Custom Webhooks Parameters
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with generic webhooks.
+
+Sematext Notification Hooks support a set of variables you can use to customize the information sent from Sematext to the destination of your choice. 
+
+The following variables are supported:
+
+ * **$webHookName** - the name of the webhook
+ * **$applicationId** - application identifier
+ * **$applicationToken** - application token (be careful about providing one as it may allow unauthorized users to send or retrieve data for an App)
+ * **$ruleType** - the type of the Alert Rule
+ * **$createTimestamp** - alert creation timestamp
+ * **$backToNormal** - flag indicating notification type that informs about the non-alerting state
```

Comment:
What sort of value(s) can this variable have? 0/1? true/false?  yes/no?  Something else?
flag indicating notification type that informs about the non-alerting state => XXXX when the notification is for the incident going back to non-alerting state; YYYY otherwise

Reply:
Added true/false.